mango-core/src/main/java/org/calrissian/mango/domain/AttributeStore.java

```diff
@@ -47,6 +37,9 @@
      */
     <T> Attribute<T> get(String key);
 
+    <T> Attribute<T> get(String key, Class<T> clazz);
```

Comment:
This is a design thats been int he attribute store for a long time when a
user has a single-valued attribute and they know that it's single valued

On Wed, Mar 11, 2015 at 2:39 AM, eawagner <notifications@github.com> wrote:

> In
> mango-core/src/main/java/org/calrissian/mango/domain/AttributeStore.java
> <https://github.com/calrissian/mango/pull/178#discussion_r26191643>:
>
> > @@ -47,6 +37,9 @@
> >       */
> >      <T> Attribute<T> get(String key);
> >
> > +    <T> Attribute<T> get(String key, Class<T> clazz);
>
> On a side note, since this is a multivalued data object, shouldn't the get
> methods return collections?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/calrissian/mango/pull/178/files#r26191643>.
>